extensions/machine-learning/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "machine-learning",
   "displayName": "%displayName%",
   "description": "%description%",
-  "version": "0.5.0",
+  "version": "0.6.0",
```

Comment:
we dont need to change azdata dependency version? its "azdata": ">=1.21.0 currently

Reply:
I think that has to be done in the marketplace